README.md

```diff
@@ -4,19 +4,34 @@ machine. This is pretty much a few packages which, for one reason or
 another, aren't the best fit for going into Homebrew itself.
  
 ## Installation
-    brew install https://raw2.github.com/leedm777/homebrew-asterisk/master/iksemel.rb
-    brew install homebrew/dupes/ncurses
-    brew install https://raw2.github.com/leedm777/homebrew-asterisk/master/asterisk.rb
+    xcode-select --install
```

Comment:
oh; wait. nevermind. it's b/c of the whole command line tools vs. xcode build thing.